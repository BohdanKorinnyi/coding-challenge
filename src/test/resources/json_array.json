[
  {
    "key0": [
      [
        2,
        "text"
      ],
      "text"
    ]
  },
  {
    "key1": 2
  },
  {
    "key2": {
      "key3": "text"
    }
  },
  {
    "key4": [
      2,
      2,
      {
        "key5": "text"
      }
    ]
  }
]