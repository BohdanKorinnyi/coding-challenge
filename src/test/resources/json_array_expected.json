[
  {
    "key0": [
      [
        "C81E728D9D4C2F636F067F89CC14862C",
        "1CB251EC0D568DE6A929B520C4AED8D1"
      ],
      "1CB251EC0D568DE6A929B520C4AED8D1"
    ]
  },
  {
    "key1": "C81E728D9D4C2F636F067F89CC14862C"
  },
  {
    "key2": {
      "key3": "1CB251EC0D568DE6A929B520C4AED8D1"
    }
  },
  {
    "key4": [
      "C81E728D9D4C2F636F067F89CC14862C",
      "C81E728D9D4C2F636F067F89CC14862C",
      {
        "key5": "1CB251EC0D568DE6A929B520C4AED8D1"
      }
    ]
  }
]